docs/devguide/debugging_guide.rst

```diff
@@ -0,0 +1,68 @@
+RackHD Debugging Guide
+-----------------------
+
+Discovery with a Default Workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sequence Diagram for the Discovery Workflow
+
+.. image:: https://www.websequencediagrams.com/cgi-bin/cdraw?lz=dGl0bGUgRGVmYXVsdCBEaXNjb3ZlcnkgV29ya2Zsb3cKClNlcnZlci0-UmFja0hEOiBESENQIGZyb20gUFhFKG5pYyBvciBCSU9TKQoAHQYtPgAtBjogSVNDACkGcmVzcG9uc2Ugd2l0aCBJUAAZEURIQ1AtcHJveHkAHQ9ib290ZmlsZQB2EVJlcXVlc3QgdG8gZG93bmxvYWQAJAkgdmlhIFRGVABWElRGVFAgc2VuZHMgcgA7BmVkIAAuBShtb25vcmFpbC5pcHhlKQpub3RlIGxlZnQgb2YgAIFJCACCBgYgbG9hZHMgACQNIGFuZCBpbml0aWF0ZXMgb24AgTUFbG9hZGVyAIIyEUlQWEUgc2NyaXB0AIB_CHMgd2hhAIFHBwCCWAYAgmoGIChodHRwAIEIB3JpZ2gAgQsFAIF8CQAeBmxvb2tzIHVwIElQIGFkZHJlc3Mgb2YgSFRUUACBXwgAgywGAG0MdG8gZmluZCB0aGUgbm9kAIIvBml0J3MgbWFjLQBABy4AYhdJZgAuCmlzIGFscmVhZHkgImtub3duIiwgaXQgd2lsbCBvbmwAg0oIZCBpADAFcmUncyBhbiBhY3RpdmUgdwCEYgcgdGhhdCdzIGJlZW4gaW52b2tlZCByZWxhdGVkIHRvAIEeCQBsJW4ndCAAgQYFAIEACmNyZWF0ZSBhAGcKKGQAhW8HAHsIKQCDQAUAgSYIAIUmBWFuIGkAgiYOAINcCACBKwUAhWERAIQGBQCDUwcoAINNBQCDQAdjYWxscyBhIFByb2ZpbGUpICh2aWEgAINRCwCEWxAAPgwAhjEFZACHEwltaWNyb2tlcm5lbACEbglyZACEIggAhyAQAIUZBQCEXQlzdGF0aWMAhW0GLQCDdwV2bWxpbnV6IABLBgCHMxEAFwgAKzMAgRIGAIISEgCBIg0AhkYidGhlAIEWBwCBYgxhbmQgdHJhbnNmZXJzIGNvbnRyb2wgKGJvb3RzAIRHBQCCGwwAhzgXdGhlAIJAF2lzIHNlAIhYBQCGOAhhbmQgbGF1bmNoIGEgTm9kZUpTIHRhc2sgcnVubm4Ah1ITAIdQCXRoZQCJQgVzdHJhcC5qcyB0ZW1wbGF0ZQCKHxEAGQ1maWxsZWQgb3UAg3kHdmFsdWVzIHNwZWNpZmljAIYIDCBiYXNlZCBvbiBhAIgHBXVwAIkpFnJ1bnMAKgcAgQULAItREQCBJAkgYXNrcyBmb3IAgWUFcwCFSwdzaG91bGQgSSBkbz8Ai2ISZGF0YSBwYWNrZQCKNQUAMgcAhU4fAIxmEiBwYXNzZXMAi1cFAHcHAIZ4BXRlcnJvZ2F0ZSBoYXJkd2FyZQpsb29wAIEgBWVhY2ggVGFzawCKNQwKICAgAIs5BwCNOgpvdXRwdQCBFwkKZW5kAIpKFwCDXQUAJwdzdG9yZWQgYXMgY2F0YWxvZ3MgaW4Aix8IAIkAKmkAizcIIGkAhTMFZmlndXJlAIhpB1NLVSBkZWYAjEQFb25zAIpEBXByb2NlAIIIBXRoZXNlAHQKdG8gZGV0ZXJtaW5lAItCBVNLVQBbGgCKcgkAYAUAhD4JAIp3CQByBWVkLACGQwkAgRkGdGludQCKeghhdCwgb3RoZXJ3aXMAbAYAjDQFAItlBWJlIHRvbACLJgVyZWJvb3QAhAlOTm90aGluZyBtb3JlLCB0aGFua3MgLSBwbGVhc2UAawcAikgMCgo&s=napkin
```

Comment:
can't really - not and use what's available, this is a dynamic link to public site (websequencediagrams)